records_mover/records/delimited/utils.py

```diff
@@ -20,11 +24,17 @@ def complain_on_unhandled_hints(fail_if_dont_understand: bool,
             logger.warning(f"Did not understand these hints: {unhandled_bindings_str}")
 
 
-def cant_handle_hint(fail_if_cant_handle_hint: bool, hint_name: str, hints: RecordsHints) -> None:
+def cant_handle_hint(fail_if_cant_handle_hint: bool,
+                     hint_name: str,
+                     hints: Union[RecordsHints,
+                                  UntypedRecordsHints,
+                                  ValidatedRecordsHints]) -> None:
+    # TODO: this probably doesn't handle ValidatedRecordsHints yet
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker